Gemfile

```diff
@@ -12,6 +12,7 @@ group :development do
   platforms :mri, :mingw do
     gem 'yard', require: false
   end
+  gem 'pry-byebug'
```

Comment:
Was this meant to be added?

Reply:
@drwl Thank you for your replying and pointing my mistakes out.
I already fixed commits in my local environment in accordance with your indication.

As you request
> I went commit by commit to try and review but found these commits to be too big for me to review:
> 28e5e84
> b7cac0a
> 7b2697e
> Would it be possible for you to split these ones up?

so I'd like to 
*   close this PR
*   make another new PR without these 3 commits
*   after the new PR is merged, make the other PR spliting up these 3 commits 

May I continue to contribute in this way?

Reply:
I'm sorry, I used this gem in my local environment and forgot to remove this from Gemfile.

Reply:
No problem :) do this often enough